awscli/arguments.py

```diff
@@ -440,11 +438,11 @@ def add_to_params(self, parameters, value):
             parameters[self._serialized_name] = unpacked
 
     def _unpack_argument(self, value):
-        service_name = self.operation_object.service.endpoint_prefix
-        operation_name = xform_name(self.operation_object.name, '-')
+        service_name = self._operation_model.service_model.endpoint_prefix
```

Comment:
Should we be using endpoint_prefix?

I have been using ``operation_model.service_model.service_name``. I was not sure to use ``endpoint_prefix`` because they do not always line up with their cli name.